tests/test_attentionunet.py

```diff
@@ -50,6 +56,19 @@ def test_attentionunet(self):
             self.assertEqual(output.shape[0], input.shape[0])
             self.assertEqual(output.shape[1], 2)
 
+    def test_attentionunet_kernel_size(self):
+        args_dict = {
+            "spatial_dims": 2,
+            "in_channels": 1,
+            "out_channels": 2,
+            "channels": (3, 4, 5),
+            "up_kernel_size": 5,
+            "strides": (1, 2),
+        }
+        model_a = att.AttentionUnet(**args_dict, kernel_size=5)
+        model_b = att.AttentionUnet(**args_dict, kernel_size=7)
+        self.assertNotEqual(get_net_parameters(model_a), get_net_parameters(model_b))
```

Comment:
I would suggest as a more useful test to store the number of expected parameters for model_a and model_b and compare against those values. With that minor change we'd be ready to go with this PR I think.

Reply:
Agreed – done